crates/iota-proxy/src/lib.rs

```diff
@@ -203,7 +203,7 @@ mod tests {
     }
 
     /// this is a long test to ensure we are timing out clients that are slow
-    #[tokio::test]
+    #[tokio::test(flavor = "current_thread", start_paused = true)] // start_paused means time is simulated, progresses without waiting
```

Comment:
why is this needed?

Reply:
this was to try and skip the wait for the timeout test. It doesn't seem to work as simple as that so I've reverted for now.